modules/openapi/test/resources/baz.json

```diff
@@ -3,6 +3,5 @@
   "info": {
     "title": "TestService",
     "version": "1"
-  },
-  "components": {}
```

Comment:
This is a change introduced by using `convertToNode` instead of `convert(...).toNode`. Mildly concerning, since it shows the generated JSON might change, but
* it's still valid OpenAPI and semantically identical
* the larger example in foo.smithy didn't change at all

Reply:
that's fine imho